package.json

```diff
@@ -110,6 +110,7 @@
         "@types/command-line-usage": "^5.0.1",
         "@types/common-tags": "^1.4.0",
         "@types/lunr": "^2.3.2",
+        "@types/mocha": "^8.2.2",
```

Comment:
Without this, my editor was having trouble finding the `describe` and `it` globals provided by mocha.